package.json

```diff
@@ -31,6 +31,7 @@
     "cytoscape-dagre": "^2.4.0",
     "graphlib": "^2.1.8",
     "lodash": "^4.17.21",
+    "moment": "^2.30.1",
```

Comment:
You probably already know that Moment has their own recommendations for alternatives: https://momentjs.com/docs/#/-project-status/recommendations/

Personally I've used [date-fns](https://date-fns.org/) and have had no issues with it. But I think that's something that can be addressed separately from this PR.

Reply:
I am having problems with date-fns and typescript, 'Intl' has no exported member named 'RelativeTimeFormatOptions'. Did you mean 'DateTimeFormatOptions'? so can I look into this later then I remove moment later. or moment has to be removed before PR. I had changed the date to use date-fns and working but that tpescript error

Reply:
Without knowing what you tried doing, the error message in isolation isn't too informative. I'm not even really strongly advocating for `date-fns`. I was just pointing out that I've used it (yes, in TypeScript projects) and thought it was fine.

But more importantly, to quote myself 😂 :

> I think that's something that can be addressed separately from this PR.